docs/docs/features/keymaps.mdx

```diff
@@ -4,7 +4,6 @@ sidebar_label: Keymaps
 ---
 
 import KeymapExample from "../keymap-example.md";
-import KeymapExampleFile from "../keymap-example-file.md";
```

Comment:
It looks like we can delete this file now.

Reply:
The file is still in use in the new shield guide. I thought about moving the file, or putting the content into the new shield guide. What are your thoughts?

Reply:
Ah, good catch thanks. I'd say let's leave it for now.